sql/date-and-time-types.md

```diff
@@ -0,0 +1,236 @@
+---
+title: date and time types
+category: user guide
+---
+
+# Date and time types
+
+TiDB supports the following data types to store temporal values: `DATE`, `TIME`, `DATETIME`, `TIMESTAMP`, and `YEAR`. Each of these types has its own range of valid values, and uses a zero value to indicate that it is an invalid value. In addition, the `TIMESTAMP` and `DATETIME` types can automatically generate new time values on modification.
+
+When dealing with date and time value types, please note: 
+
++ Although TiDB tries to interpret different formats, the date-portion must be in the format of year-month-day (for example, '1998-09-04'), rather than month-day-year or day-month-year.
++ If the year-portion of a date is specified as 2 digits, TiDB converts it based on the following rules:
+  - Values between 70 and 99 is converted to a value between 1970 and 1999
+  - Values between 00 and 69 is converted to a value between 2000 and 2069
++ If a numberic value is needed in the context, TiDB automatically converts the date or time value into a numeric type. For example,
+
+  ```sql
```

Comment:
Please indent the code block with four spaces, to avoid possible display errors on the PingCAP website.